programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -122,6 +122,25 @@
   - Jonah Kowall (@jkowall, jkowall@kowall.net)
 - Upstream Issue: https://github.com/jaegertracing/jaeger/issues/5084
 
+### OpenTelemetry
+
+#### One Logging Bridge per Language 
+
+- Description: One of the goals set for the OpenTelemetry project in 2024 is to have at least one logging bridge per Language SIG, so that our end-users can start using OTLP Logging natively in their applications. While some languages have such a bridge already, some have the desire to implement at least one bridge but are lacking the engineering resources to do so. This internship starts by taking a look at the current state, marking which languages have a bridge already and which are lacking. The next step is to propose and implement at least one bridge for at least one language that doesn't have such a bridge yet.
+- Expected Outcome: At least one logging bridge is implemented for at least one language.
+- Recommended Skills: One (or more!) of the missing languages supported by OpenTelemetry
+  - C++
+  - Erlang
+  - Go
+  - JavaScript
+  - PHP
+  - Ruby
+  - Swift
+- Mentor(s):
+  - Juraci Paixão Kröhling (@jpkrohling, juraci.kroehling@grafana.com)
```

Comment:
We strongly encourage mentors to have a backup mentor. Do you think you can find one?

Reply:
Yes!